gemfile.d/lock.rb

```diff
@@ -0,0 +1 @@
+gem "foreman_remote_execution", "<= 12.0.5"
```

Comment:
What's wrong with 12.0.6? Wouldn't semver suggest something like `< 13` would be more appropriate?